lib/cube_browser/__init__.py

```diff
@@ -539,26 +539,35 @@ def __init__(self, plots):
 
         self._slider_by_name = {}
         self._name_by_slider_id = {}
+        if self._axis_by_name:
+            name_len = max([len(name) for name in self._axis_by_name])
+        children = []
         for axis in self._axis_by_name.values():
             if hasattr(axis, 'coord') and axis.coord.units.is_time_reference():
-                opts = [(axis.coord.units.num2date(axis.coord.points[i]), i)
-                        for i in range(axis.size)]
+                pairs = [(axis.coord.units.num2date(axis.coord.points[i]), i)
+                         for i in range(axis.size)]
             elif hasattr(axis, 'coord'):
-                opts = [(axis.coord.points[i], i) for i in range(axis.size)]
+                pairs = [(axis.coord.points[i], i) for i in range(axis.size)]
             else:
-                opts = [(i, i) for i in range(axis.size)]
-            options = OrderedDict(opts)
-            slider = ipywidgets.SelectionSlider(description=axis.name,
-                                                options=options)
+                pairs = [(i, i) for i in range(axis.size)]
+            options = OrderedDict(pairs)
+            slider = ipywidgets.SelectionSlider(options=options)
             slider.observe(self.on_change, names='value')
             self._slider_by_name[axis.name] = slider
             self._name_by_slider_id[id(slider)] = axis.name
+            # Explicitly control the slider label in order to avoid
+            # fix width widget description label wrapping issues.
+            # XXX: Adjust px/em scale-factor dynamically based on font-size.
+            scale_factor = .65
```

Comment:
@bjlittle no, just naming it does not make it less magic :wink: As this is a px to em conversion factor, it assumedly means you, in the loosest sense, have defined the pixel width of 1em in the CSS somewhere. Any chance you can hook into that?? Otherwise the comment above does help, even if it doesn't give the reason for the value!